src/NServiceBus.Core/NServiceBus.Core.csproj

```diff
@@ -14,7 +14,6 @@
 
   <ItemGroup Condition="'$(TargetFramework)' == 'net472'">
     <Reference Include="System.Transactions" />
-    <Reference Include="System.Configuration" />
```

Comment:
I'm assuming the build would fail if we had any other code that tries to access dependencies from `System.Configuration`. Is that correct or do we need to double check?

Reply:
There is some weirdness in Core around some references that are related to the licensing package, but I verified that everything is okay here.

Reply:
:shipit: